rocky/reports/views/aggregate_report.py

```diff
@@ -193,6 +198,7 @@ def save_report(self) -> ReportOOI:
             parent=None,
             has_parent=False,
             observed_at=observed_at,
+            name=report_name[0][1],
```

Comment:
This will crash if report_name is empty or missing 2 items. It will be also more readable if report_name would be report names, it is a collection of names.